core/repository/manager/src/main/java/org/eclipse/rdf4j/repository/manager/RemoteRepositoryManager.java

```diff
@@ -218,13 +186,8 @@ public RepositoryConfig getRepositoryConfig(String id) throws RepositoryExceptio
 			protocolSession.setUsernameAndPassword(username, password);
 
 			int serverProtocolVersion = Integer.parseInt(protocolSession.getServerProtocol());
```

Comment:
Is this variables used anymore? Should we throw an exception if the version is `< 10`?